docker/functions/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM node:14.16.0
 
 WORKDIR /usr/src/app
 
-RUN npm i -g azure-functions-core-tools@3 --unsafe-perm true
+RUN npm i -g azure-functions-core-tools@4 --unsafe-perm true
```

Comment:
I think this change needs to be done also in `docker/functions/Dockerfile` file?